terraform/tutorial.md

```diff
@@ -0,0 +1,351 @@
+# Cloud Operations Sandbox Tutorial
+
+## Overview
+
+The Cloud Operations Sandbox is intended to make it easy for you to deploy and run a non-trivial application that lets you explore the Google Cloud Platform services, particularly the [Ops Management](http://cloud.google.com/products/operations) (formerly Stackdriver) product suite. Ops Management is a suite of tools that helps you gain full observability into your code and applications.
+
+The Hipster Shop application used in the sandbox is intended to be sufficiently complex such that you can meaningfully experiment with it, and the Sandbox automatically provisions a new demo cluster, configures and deploys Hipster Shop, and simulates real users.
+
+With the Sandbox running, you can experiment with various Ops Management tools to solve problems and accomplish standard SRE tasks in a sandboxed environment without impacting your production monitoring setup.
+
+### Architecture of the Hipster Shop application
+
+The Hipster Shop application consists of a number of microservices, written in a variety of languages, that talk to each other over gRPC.
+
+**Note:** We are not endorsing this architecture as the best way to build a real online store. This application is optimized for demonstration and learning purposes.  It illustrates a large number of cloud-native technologies, uses a variety of programming languages, and provides an environment that can be explored productively with Ops Management tools.
+
+The Git repository you cloned has all the source code, so you can explore the implementation details of the application. See the repository [README](https://github.com/GoogleCloudPlatform/stackdriver-sandbox) for a guided tour.
+
+### Prerequisites
+
+You must have an active Google Cloud Platform Billing Account. If you already have one, you can skip this section.
+
+Otherwise, to create a GCP Billing Account, do the following:
+
+1. Go to the Google Cloud Platform [Console](https://console.cloud.google.com/) and sign in (if you have an account), or sign up (if you don't have an account).
+1. Select **Billing** from the navigation panel and follow the instructions.
+
+For more information, see ["Create a new billing account"](https://cloud.google.com/billing/docs/how-to/manage-billing-account).
+
+## Set up
+
+The installation process takes a few minutes. When it completes, you see a message like the following:
+
+```bash
+********************************************************************************
+Cloud Operations Sandbox deployed successfully!
```

Comment:
This seems to render weirdly on the tutorial with artificial `\` line breaks.  I'm not really sure if there's a good solution to this though. I wonder if it could look more natural to break the lines yourself?

Reply:
I'll look into it!